src/containers/Review/ReviewSubmit.tsx

```diff
@@ -150,6 +165,17 @@ const ReviewSubmitButton: React.FC<ReviewSubmitProps & ReviewSubmitButtonProps>
       return
     }
 
+    // Check MISMATCH previous (when is Final Decision)
+    if (
+      assignment.isFinalDecision &&
+      !!previousAssignment &&
+      previousAssignment.review?.reviewDecision?.decision !== getDecision()
+    ) {
+      // Should submit when user clicks OK
+      showDecisionMismatchWarning()
+      return
+    }
+
```

Comment:
We've had this problem before -- there's no way to close the modal without clicking "OK". Needs an "X" in the corner. :)

Reply:
Ooppsy! Fixed now.

This modals... they are terrible. I'm not sure how we can improve this, but shouldn't be too difficult. I'll see if Andrei has some better ideas on what we can do here. Which can be done on issue #308

Reply:
Hey, thanks, but it doesn't work 🙁
![Screen Shot 2021-08-05 at 3 40 49 PM](https://user-images.githubusercontent.com/5456533/128287385-e0665f9a-749f-4ec2-8897-ba3f75bdf8a2.png)
I see the icon -- but clicking it does nothing. (This is only for Final Decision when disagreeing with previous. Seems to be fine in other situations)

Reply:
Greeeeat catch! Wow, that was a long PR. But it's now fixed 5d1d875 : ) Thanks